core/shared/src/main/scala/clue/ApolloClient.scala

```diff
@@ -78,16 +77,16 @@ class ApolloClient[F[_], S, CP, CE](
   state:                Ref[F, State[F, CP]],
   connectionStatus:     SignallingRef[F, PersistentClientStatus]
 )(implicit
-  F:                    ConcurrentEffect[F],
+  F:                    Async[F],
   backend:              PersistentBackend[F, CP, CE],
-  timer:                Timer[F],
+  timer:                Temporal[F],
```

Comment:
It seems `Async` already extends `Temporal`. We don't want the `timer` parameter then.

Reply:
Good catch